docs/whats-news/2023/2023-06.md

```diff
@@ -17,3 +17,11 @@ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/mozilla/a
 ```
 
 and further details are at [https://experimenter.info/nimbus-cli](/nimbus-cli).
+
+## Jetstream
+
+### Computing the date for expected results
+We are now computing the date that the Jetstream results will be available for a given experiment. This new value is based on enrollment end date (or proposed enrollment end date if enrollment is not yet ended).
+
+![expected_results](https://github.com/mozilla/experimenter-docs/assets/43795363/51754bcd-4e1d-4b1a-a612-9aeeb526e0ae)
+
```

Comment:
```suggestion

## Experimenter

### Release date for First Run experiments

Our approach for calculating dates and duration for experiments did not previously take the release trains into account. For example, a first run experiment targeting Firefox desktop v123 might be launched (status=Live) on 1/2/2023. However, v123 might not be released until 1/15/2023. The start_date for this experiment will therefore be 1/2/2023, but users will not start seeing it until 1/15 when the version is released. 

We added another date, "proposed release date" or "first run release date", for first-run mobile experiments which will represent the [release date of the browser version](http://whattrainisitnow.com/) that is being targeted by the user. 

This date gives us the ability to:
* Correctly compute enrollment and proposed end dates based on when users will actually start getting the release version
* Send reminder emails to experiment owners at the right time so that they can end enrollment/end experiments

This new field shows up on the Audience page of the experiment, and is also viewable on the Summary page and on the timeline:

<img width="1677" alt="release_date_audience" src="https://github.com/mozilla/experimenter-docs/assets/43795363/2ac5e65b-5f0d-4e0d-8313-c7a71f280bc3">

<img width="1625" alt="release_date_timeline" src="https://github.com/mozilla/experimenter-docs/assets/43795363/65e081c2-c283-4f25-a1ba-bd19ccb9d656">

<img width="1525" alt="release_date_summary" src="https://github.com/mozilla/experimenter-docs/assets/43795363/6ba10e60-8a12-4fbe-8035-e6d36f7caade">


## More...
```